product/views/ManageIQ_Providers_AnsibleTower_ConfigurationManager.yaml

```diff
@@ -15,7 +15,7 @@ title: Ansible Tower Providers
 name: ConfigurationManagerForeman
```

Comment:
Not letting it block this PR as this PR is blocking a bunch, but that looks like a bug.